airbyte-cdk/bulk/toolkits/load-gcs/src/main/kotlin/io/airbyte/cdk/load/file/gcs/GcsClientFactory.kt

```diff
@@ -58,7 +58,6 @@ class GcsClientFactory(
                                         )
                                 },
                                 assumeRoleCredentials = null,
-                                null
```

Comment:
GCS client calls the S3 client, so we need to drop the `s3ClientConfig: S3ClientConfigurationProvider?` param here too